api.md

```diff
@@ -403,7 +403,7 @@ still work on the resulting hypertable.
 
 |Name|Description|
 |---|---|
-| `main_table` | Identifier of table to convert to hypertable |
+| `relation` | Identifier of table to convert to hypertable |
```

Comment:
Assuming that punctuation is our standard:

```suggestion
| `relation` | Identifier of table to convert to hypertable. |
```

Reply:
Is there a reason you don't just use "table" here?  I mean, I guess `relation` is correct from the format database semantics perspective, but I think will be confusing for the average user....

Reply:
> Is there a reason you don't just use "table" here? I mean, I guess `relation` is correct from the format database semantics perspective, but I think will be confusing for the average user....

We had a discussion in the team and decided to use `hypertable` when it was a hypertable, and `relation` for other cases (e.g., a Postgres table, hypertable or cagg, etc.)